pkg/daemon/update.go

```diff
@@ -419,6 +419,12 @@ func calculatePostConfigChangeActionFromFileDiffs(diffFileSet []string) (actions
 		imageRegistryAuthFile,
 		"/var/lib/kubelet/config.json",
 	}
+	directoriesPostConfigChangeActionNone := []string{
+		// We probably don't want to use this exact path because NMState is
+		// planning to add a service that applies configs from it too, and we
+		// want to make sure our service is the only one processing the configs.
+		"/etc/nmstate",
```

Comment:
I should change this to be /etc/nmstate/openshift. We don't necessarily want to ignore _everything_ in /etc/nmstate.